cardinal/testutils/test_utils.go

```diff
@@ -188,8 +212,13 @@ func MakeWorldAndTicker(t *testing.T, opts ...cardinal.WorldOption) (world *card
 
 	startGameOnce := sync.Once{}
 	// Create a function that will do a single game tick, making sure to start the game world the first time it is called.
-	doTick = func() {
+	doTick = func(timeoutTime ...int) {
```

Comment:
For this one, either make it a required time.Duration or a boolean that disables the timeout.

Alternatively, remove the timeout portion of this doTick method completely. Debugging tests without having to worry about this method forcing a test failure sounds useful, and if we find that tests in CI or locally are hanging instead of failing we can start using the `-timeout` flag for tests.